pkg/cmd/clustertask/start.go

```diff
@@ -130,7 +130,7 @@ like cat,foo,bar
 	c.Flags().BoolVarP(&opt.Last, "last", "L", false, "re-run the clustertask using last taskrun values")
 	c.Flags().StringSliceVarP(&opt.Labels, "labels", "l", []string{}, "pass labels as label=value.")
 	c.Flags().BoolVarP(&opt.ShowLog, "showlog", "", false, "show logs right after starting the clustertask")
-	c.Flags().Int64VarP(&opt.TimeOut, "timeout", "t", 3600, "timeout for taskrun in seconds")
+	c.Flags().StringVar(&opt.TimeOut, "timeout", "1h", "timeout for taskrun (default \"1h\")")
```

Comment:
My apologies for not explaining this better:

You actually don't need to include `(default \"1h\")` in the description. Cobra will add that piece to the description since you have specified the `1h` default.

Reply:
Thanks @danielhelfand now I got it. I've just updated the code with your suggestion.